RecCalorimeter/src/components/CorrectCaloClusters.h

```diff
@@ -118,6 +138,19 @@ class CorrectCaloClusters : public GaudiAlgorithm {
                           int systemID,
                           int layerID);
 
+  /**
+   * Get sum of energy from cells in the whole calorimeter.
+   * This energy is not calibrated.
+   *
+   * @param[in]  cluster       Pointer to cluster of interest.
+   * @param[in]  readoutName   Name of the readout.
```

Comment:
systemID is also input parameter for this function.

Reply:
added in the latest commit